common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/azureactivedirectory/AzureActiveDirectoryAuthorizationResultFactory.java

```diff
@@ -103,11 +103,21 @@ public AzureActiveDirectoryAuthorizationResult createAuthorizationResult(final i
                 break;
 
             case AuthenticationConstants.UIResponse.BROWSER_CODE_DEVICE_REGISTER:
-                Logger.verbose(TAG, "Device needs to have broker installed, we expect the apps to call us"
-                        + "back when the broker is installed");
+                Logger.info(TAG, "Device Registration needed, need to start WPJ");
                 result = createAuthorizationResultWithErrorResponse(AuthorizationStatus.FAIL,
-                        MicrosoftAuthorizationErrorResponse.AUTHORIZATION_FAILED,
-                        MicrosoftAuthorizationErrorResponse.BROKER_NEEDS_TO_BE_INSTALLED);
+                        MicrosoftAuthorizationErrorResponse.DEVICE_REGISTRATION_NEEDED,
```

Comment:
Why are we passing the same parameter twice?

Reply:
The pattern I'm seeing is that the error code itself (MicrosoftAuthorizationErrorResponse.XXXX) can also be used as a error message.

We could also have separate values for this one.